src/services/chill_other.ts

```diff
@@ -62,9 +62,11 @@ async function buildChillTxs(api: ApiPromise, threshold: BN, chillThreshold: Per
 	allNominators.sort((n1, n2) => n1.stake.cmp(n2.stake));
 	// filter those that are below
 	const toRemoveAll = allNominators.filter((n) => n.stake.lt(threshold));
-	const ejectedStake = toRemoveAll
-		.map(({ stake }) => stake)
-		.reduce((prev, current) => prev = current.add(prev));
+	const ejectedStake = toRemoveAll.length
```

Comment:
what's the benefit of this? if `toRemoveAll` is empty, the map will also return `0` right?

Reply:
We get the following error with reduce https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce#exceptions